### DISCUSSION TITLE
Shared folder

### DISCUSSION THREAD

**Discussion by kastony** [masbug/flysystem-google-drive-ext#58] [General]
Hi,
how can I have access to shared folders ; 

Thanks

**Comment by fredsal** [comment]
>how can I have access to shared folders 

Reading documentation will give you the answer

**Comment by erikn69** [comment]
With the path, same as regular folder

**Comment by alepap** [comment]
but is it enough to set GOOGLE_DRIVE_TEAM_DRIVE_ID in the .env file?
I have no problem creating directories and subdirectories within a folder in "my-drive" but when I set GOOGLE_DRIVE_TEAM_DRIVE_ID I get "Unable to read file from location: MyFolder. File not found.

In the example "MyFolder" is the folder I want to create dynamically.

Thanks in advance
